### ISSUE TITLE
server mode doesn't work probably 

### ISSUE THREAD

**Issue by loewechiu** [SAP/PyRFC#98]
Hi Gurus,

I just test with PyRFC/examples/serverStfcConnection.py , and find it can't register to sap gateway normally at all.

First,
my cfg is like:
```ini
gwhost = xxx.xxx.xx.xx
ghserv = sapgw10
```
it just go through 3300 not 3310 and return error:

```shell
pyrfc._exception.CommunicationError: RFC_COMMUNICATION_FAILURE (rc=1): key=RFC_COMMUNICATION_FAILURE, message=
LOCATION    CPIC (TCP/IP) on local host ITCNSUZM01294 with Unicode
ERROR       partner '153.95.192.18:3300' not reached
TIME        Mon Apr 29 10:20:23 2019
RELEASE     753
COMPONENT   NI (network interface)
VERSION     40
RC          -10
MODULE      D:/depot/bas/753_REL/src/base/ni/nixxi.cpp
LINE        3449
DETAIL      NiPConnect2: 153.95.192.18:3300
SYSTEM CALL connect
ERRNO       10061
ERRNO TEXT  WSAECONNREFUSED: Connection refused
COUNTER     2
 [MSG: class=, type=, number=, v1-4:=;;;]
```

Then I changed my cfg to:
my cfg is like:

```ini
gwhost = xxx.xxx.xx.xx
ghserv = sapgw10
sysnr = 10
```

it successfully appeared in sap gateway, however, it keeps flashing (the number/id in gateway list keeps change in mileseconds),
So can't be used at all?

Could any one help me ?

Thank you very much.

**Comment by bsrdjan** [comment]
I am sorry but the server functionality is currently not covered by PyRFC support, only the client.

**Comment by bsrdjan** [comment]
Server bindings are implemented in main branch: df9805e

More info: https://github.com/SAP/PyRFC/issues/193#issuecomment-720645607

**Comment by bsrdjan** [comment]
Server bindings are published with 2.3.0 release on PyPI, feel free to test.
